### DISCUSSION TITLE
Extraction of segmentation model to onnx and use case example

### DISCUSSION THREAD

**Discussion by baril-jf** [roboflow/notebooks#141] [Q&A]
I have followed the How to Train YOLOv7 Instance Segmentation on a Custom Dataset notebook for a special case and it works really well.  I now want to export the model to onnx format and use it in that format to produce the same output of the predict method from a .jpg image (ideally from a folder containing many .jpg files but 1 image will do) and I have a hard time achieving this.  Would you have an example of this use case ?  Thanks !

**Comment by SkalskiP** [comment]
Hi @baril-jf 👋🏻! Sorry for the late reply, I was quite busy. What part of the task seems to be the problem? Conversion to ONNX?
